logicrunner/logicrunner.go

```diff
@@ -149,6 +149,8 @@ func NewLogicRunner(cfg *configuration.LogicRunner) (*LogicRunner, error) {
 	return &res, nil
 }
 
+func (lr *LogicRunner) LRI() {}
```

Comment:
What's purpose of LRI? To distinguish LR and OS/ES?

Reply:
To distinguish LogicRunner in out DI from other classes that have OnPulse